wls-gui-wahllokalsystem/tests/components/common/dialogs/BaseDialogBegruendung.spec.ts

```diff
@@ -0,0 +1,105 @@
+import {
+  COMPONENT_EVENT_TESTS,
+  COMPONENT_RENDER_TESTS,
+  getSnapshotFilename,
+} from "@tests/utils/testutils.ts";
+import { mount, VueWrapper } from "@vue/test-utils";
+import { afterEach, describe, expect, it, vi } from "vitest";
+
+import BaseDialogBegruendung from "@/components/common/dialogs/BaseDialogBegruendung.vue";
+import vuetify from "@/plugins/vuetify.ts";
+
+describe("BaseDialogBegruendung.vue", () => {
+  let wrapper: VueWrapper;
+  vi.stubGlobal("visualViewport", new EventTarget());
+  const ResizeObserverMock = vi.fn(() => ({
+    observe: vi.fn(),
+    unobserve: vi.fn(),
+    disconnect: vi.fn(),
+  }));
+  vi.stubGlobal("ResizeObserver", ResizeObserverMock);
+
+  const BEGRUENDUNG = "Begründung";
+
+  afterEach(() => {
+    if (wrapper) wrapper.unmount();
+  });
+
+  describe(COMPONENT_RENDER_TESTS, () => {
+    it("should_showBaseDialogBegruendung_WithDisabledSaveButton", async (context) => {
```

Comment:
entsprechend der naming convention
```suggestion
    it("should_showBaseDialogBegruendungWithDisabledSaveButton_when_mounted", async (context) => {
```

Reply:
[ae810bd](https://github.com/it-at-m/Wahllokalsystem/pull/1455/commits/ae810bd5f8cfc7b9e39b5fa008dc0850eba1ab79)